### ISSUE TITLE
Initial chat messages can be displayed (or received?) twice

### ISSUE THREAD

**Issue by marco-stephan** [ppy/osu#1230]
Image: https://puu.sh/xvBK0/3798421c01.png

**Comment by EriiYenn** [comment]
## Dear (Almighty) Developers and Contributors

**Is the Issue solved?**
It was opened a while ago an I didn't find any completed **PR** about this topic.
I'll take a look on it, if you confirm, it's still active to solve.
#
*Best regars,*
**EriiYenn**

**Comment by MrTheMake** [comment]
I myself didn't experience this issue again, but I also stopped contributing a fair while back.

**Comment by peppy** [comment]
It's safe to say this is still an issue as it's still open.

**Comment by EriiYenn** [comment]
Code seems without problems now!
I tested send **Direct Mesages** and **#lazer chat** in-game and it was without any problems.

Maybe time will tell more, if we missed something :)

**Comment by peppy** [comment]
Can you stop? Please go into read-only mode for the time being else i will have to block you from this repository. You are not being helpful even though you a trying to be.

**Comment by Aergwyn** [comment]
I had that happen to me a few minutes ago. Sadly I realized that too late so the only logs I could somehow restore (because it was still open) was the `network.log`.

[network.log](https://github.com/ppy/osu/files/2378712/network.log)

To describe what happened:
I opened the chat and the #lazer-channel wouldn't show anything at all. After nothing happening for a few seconds I switched to #osu where it would take really long to load but I could at least see a loading circle and I just waited. After a while everything appeared and further switching displayed messages in all channels. I realized then that the chat in #lazer was showing twice (as a block that repeats). Sadly I didn't check the others for the same issue.
